tests/unit/aiplatform/test_custom_job.py

```diff
@@ -661,3 +661,36 @@ def test_create_from_local_script_with_all_args(
         assert (
             job._gca_resource.state == gca_job_state_compat.JobState.JOB_STATE_SUCCEEDED
         )
+
+    def test_check_custom_job_availability(self, get_custom_job_mock, create_custom_job_mock):
```

Comment:
We're moving towards using `@pytest.mark.usefixtures` when we don't consume the fixture in the test directly (asserting some state against it) in the test:

```suggestion
    @pytest.mark.usefixtures('get_custom_job_mock', 'create_custom_job_mock')
    def test_check_custom_job_availability(self):
```

Reply:
Done